### PR TITLE
Update readme file with latest installation packages

### PR DESCRIPTION
**Pull request by brandonabela**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,10 +15,8 @@ Features:
 - [ ] Algorithm for game loop.
 
 Installing Packages using conda:
-- conda install -c anaconda pip
-- conda install -c selenium
-- conda install -c conda-forge opencv
+- conda install selenium
```
Comment (KeithGeorgeCiantar) [inline README.md]: This doesn't match with the commands given in the Anaconda website: https://anaconda.org/conda-forge/selenium
Does it still work?

**Review by KeithGeorgeCiantar — COMMENTED** [review]
Seems good, only one comment given.

File: README.md (hunk 1 above)
Comment (brandonabela) [reply README.md]: Unfortunately, "**conda install -c selenium**" does not work. It should either be "**conda install -c conda-forge selenium**" or "**conda install selenium**" for the package to install.

**Review by KeithGeorgeCiantar — APPROVED** [review]
Everything seems good.

**Pull request merged**